shared/common-adapters/text.desktop.js

```diff
@@ -317,6 +318,11 @@ export const styles = {
     ...globalStyles.fontSemibold,
     ...sizeGroups['14']
   },
+  textBodySmallBold: {
```

Comment:
I think because the "•" is Lato Bold, it's a mistake on our end. I'm guessing Zeplin takes the style of the 1st character. Let's do regular body small!